governance/paper-process.md

```diff
@@ -0,0 +1,235 @@
+# Process for creating papers
+
+This document is intended to provide a consistent mechanism for cloud native
+environmental sustainability to produce community papers, ensure they are reviewed, and subsequently
+published.  Content of this document does not supersede existing processes and
+is intended to be used in conjunction with [existing proposal to project
+instructions](process.md).
+
+## Proposal
+
+If a proposal is made that includes a paper as a deliverable, the proposal needs
+to ensure that there is a clearly identified lead and a well defined paper
+scope.
+
+The paper scope and topic should be raised in at least one WG meeting to
+solicit more volunteers (ideally 4).  Interested parties should meet at least
```

Comment:
I would add something in there, with volunteers, that we try to choose from a range of companies/geos/et cetera, to make sure the proposal takes into account more breadth.